src/libraries/Native/build-native.cmd

```diff
@@ -93,6 +97,7 @@ goto :SetupDirs
 echo Commencing build of native components
 echo.
 
+if /i "%__BuildArch%" == "wasm" set __sourceDir=%~dp0..\Unix
```

Comment:
It is not clear to me how is this change related to the build using ninja

Reply:
The refactoring of the `gen-buildsys` scripts so that we use a common one for Windows moved this line out of the libraries version of the `gen-buildsys` script and up to `build-native.cmd`. I refactored the scripts so that we have a common entry point for generating the CMake project for CoreCLR, Libraries, and Host (same as on Linux) since that makes handling determining the CMake parameters for the generator easier.